test/test-fs.c

```diff
@@ -104,6 +105,9 @@ static uv_fs_t close_req;
 static uv_fs_t mkdir_req;
 static uv_fs_t mkdtemp_req1;
 static uv_fs_t mkdtemp_req2;
+static uv_fs_t mkstemp_req1;
+static uv_fs_t mkstemp_req2;
+static uv_fs_t mkstemp_req3;
```

Comment:
Small nit, but I think you could reorganize the test code a bit to only use one or two `mkstemp_req` variables.